programs/ziti-edge-tunnel/netif_driver/linux/resolvers.c

```diff
@@ -372,10 +298,23 @@ bool try_systemd_resolved(void) {
 
 void dns_update_systemd_resolved(const char* tun, const char* addr) {
     int r;
+    struct in_addr inaddr;
     int ifindex;
 
     _cleanup_(sd_bus_flush_close_unrefp_f) sd_bus *bus = NULL;
 
+    // dbus 'ay' encodes 'array of bytes'
+    unsigned char ay[4];
+
+    r = inet_pton(AF_INET, addr, &inaddr);
+
+    if (r != 1) {
+        ZITI_LOG(ERROR, "Failed to translate dns address");
```

Comment:
will it be clear in the logs which dns address couldn't be translated? imo `addr` should be logged with this message.